packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -181,12 +181,59 @@ enum ListTileControlAffinity {
 ///
 /// {@tool sample}
 ///
-/// Here is a simple tile with an icon and some text.
+/// This example contains the different ways that [ListTile]s can be used.
```

Comment:
Maybe: This example uses a [ListView] to demonstrate different configurations of [ListTile]s in [Card]s.

This example looks great BTW.